.travis.yml

```diff
@@ -142,22 +140,15 @@ install:
       cp "$HOME/hombebrew_cache/libtorrent-rasterbar-1.0.11+git20172003.8736a59adc.el_capitan.bottle.tar.gz" "$(brew --cache)"
       brew install "$HOME/hombebrew_cache/libtorrent-rasterbar.rb"
 
+      brew install qt
+      brew link --force qt
+
       if [ "$build_system" = "cmake" ]; then
         brew unlink cmake
```

Comment:
@evsh 
why unlink and install cmake? Are you trying to install a newer version?